dev/devicelab/lib/framework/utils.dart

```diff
@@ -498,3 +500,34 @@ int parseServicePort(String line) {
   final Match match = _kObservatoryRegExp.firstMatch(line);
   return match == null ? null : int.parse(match.group(2));
 }
+
+/// If FLUTTER_ENGINE environment variable is set then we need to pass
+/// correct --local-engine setting too.
+void setLocalEngineOptionIfNecessary(List<String> options, [String flavor]) {
+  if (Platform.environment['FLUTTER_ENGINE'] != null) {
+    if (flavor == null) {
+      // If engine flavor was not specified explicitly then scan options looking
+      // for flags that specify the engine flavor (--release, --profile or
+      // --debug). Default flavor to debug if no flags were found.
+      const Map<String, String> optionToFlavor = const <String, String>{
+        '--release': 'release', '--debug': 'debug', '--profile': 'profile'
```

Comment:
nit: use one line per key:value pair so it's easier to read them.

Reply:
Done